java-manta-it/src/test/java/com/joyent/manta/client/MantaClientDirectoriesIT.java

```diff
@@ -55,6 +55,8 @@ public void beforeClass() throws IOException {
         config = new IntegrationTestConfigContext();
         mantaClient = new MantaClient(config);
         testPathPrefix = IntegrationTestConfigContext.generateBasePath(config, this.getClass().getSimpleName());
```

Comment:
Would it be possible to modify the `IntegrationTestConfigContext.generateBasePath` method rather than stripping the last character? Alternatively, another method called `IntegrationTestConfigContext.generateBasePathWithoutSeparator` (or some better name) that omits the seperator.

Reply:
I did that originally, but that method is used at various places, for eg: ```MantaClientAuthenticationChangeIT``` and in most cases the '/' at the end of the testPathPrefix is required. So for such cases, we would have to add a SEPARATOR. So I left it there and did the stripping. But yeah, another method in the ```IntegrationTestConfigContext``` sounds good. Will add that change.

Reply:
Added a new method and removed the code that strips the extra '/' from the testPathPrefix.